### PR TITLE
Added a configuration page

### LINKED ISSUE #33: Instead of using environment variables, ask the user to configure the dashboard on first load
At the moment we load the details for the account etc from the environment variables, this means that these details must be configured before compiling and deploying the site.

It would be better if we could deploy the same site to any host and allow users to enter their VSTS account name, token and project when a client first visits the page.

This could be then cached locally for subsequent loads and re-configured at any time.

The site could then allow the user to load directly into a project using routing, e.g. visiting `http://localhost:3000/{Project}/builds` would show them the dashboard for that specific project and re-use the saved authentication details.

### PR DESCRIPTION
**Pull request by MarkGaze**
Created a user interface configuration rather than the environment variables that we were using before

Fixes #33

### CODE REVIEW

**Comment by MarkGaze** [comment]
I'll have a look at the code coverage report and see if I can get some more tests in

**Comment by MarkGaze** [comment]
Also, testing the docker image as it should work now, if it does then I'll add the instructions to the README

File: README.md [hunk 1]
```diff
@@ -13,14 +13,28 @@ A dashboard to show the status of Visual Studio Team Service builds
...
+make docker-build
+```
+
+* When the image is built, run the container with:
+
+```bash
+docker run -d --name vsts-dashboard -p 8080:80 vsts-dashboard
+```
+
+* Then navigate to http:localhost:8080 and configure the dashboard with your Visual Studio Team Services details
```
Comment (benmatselby) [inline README.md]: Missing // in the URL. Can we make it a clickablemlink too?

**Comment by benmatselby** [comment]
This is good, nice to get rid of the environment variables

**Pull request merged**